packages/tools/devtools/devtools-view/src/components/data-visualization/FluidHandleView.tsx

```diff
@@ -38,6 +38,10 @@ export function FluidHandleView(props: FluidHandleViewProps): React.ReactElement
 
 	const [visualTree, setVisualTree] = React.useState<FluidObjectNode | undefined>();
 
+	// Editing:
```

Comment:
Are you still intending to do this before merging?

Reply:
At the moment the only situation it would matter is for SharedCell I think. And even then it would be when it is a value node so I don't know when freezing the tree would be helpful.

Do you think it should happen?

Reply:
Oh, right, I guess that's a tree-specific thing. No, I don't think we need that. We should probably clear out these comments, then.